Replication Project_Final.Rmd

```diff
@@ -0,0 +1,860 @@
+of state institutions, they are also farther from medical services. [...] I therefore introduce the number of medical doctors in the municipality per 10,000 residents according to the 1990 census as a predictor of homicide rates in all the regression models.""
+
+*Data were added to the dataset for analysis in order of data source - i.e. all data from the standard census first, then population density (which requires map data about the area of municipalities), then homicides, then the dummy variable. Data drawn from the Agricultural Census were added next, as they required matching by name instead of municipality code due to the format of the raw data. Once these data were combined into the main analytical dataframe, all relevant variables were computed, and then elevation data was added. Data regarding the state of Oaxaca were separated out through this process, as this state counts municipalities differently - each town is counted as its own municipality, which would skew the data. As noted above, Oaxacan municipalities are typically grouped into 30 districts for analysis--we completed this operation after creating the main analytical dataframe, recalculated all variables for the Oaxacan districts, and merged the two dataframes by municipality code.*
+
+
+```{r, warning=FALSE, error=FALSE}
+library(ggplot2)
+library(dplyr)
+library(xlsx)
+library(zoo)
+library(foreign)
+library(tidyr)
+library(gdata)
+library(maptools)
+library(gpclib) 
+library(RColorBrewer)
+library(scales)
+library(SDMTools)
+```
+
+**Municipalities, Indigenous Language-Speakers, Illiteracy**
+```{r, warning=FALSE, error=FALSE}
+#load main 1990 censo database from inegi to the town level
+censo90 <- read.table("data/censo_1990_37_var.txt", header = TRUE, sep = "\t", encoding = "latin1")
+
+#create table with variables of interest and population
+censo.1 <- tbl_df(censo90) %>%
+  filter(mun !=0 ) %>%
+  select(state = entidad, mun, twn = loc, total.pop = p_total, no.literacy1 = analfbet,
+         no.literacy2 = pob_no_l, n_hab_esp, habla_esp, state_name = nom_ent, mun_name = nom_mun) %>%
+  mutate(indi = as.numeric(as.character(n_hab_esp)) + as.numeric(as.character(habla_esp)), 
+         muncode = (state * 1000) + mun, no.literacy1 = as.numeric(as.character(no.literacy1)), 
+         no.literacy2 = as.numeric(as.character(no.literacy2)), no.literacy = no.literacy1 + no.literacy2) %>%
+  filter(muncode > 1000) %>%
+  select(muncode, twn:indi, no.literacy, -state, -mun, -no.literacy1:-habla_esp)
+
+#create variable for population in towns of less than 2500 population
+mun.total <- censo.1 %>%
+  filter(twn == 0)
+
+less.2500 <- censo.1 %>%
+  filter(twn != 0 & twn != 9998 & twn != 9999, total.pop < 2500) %>%
```

Comment:
I didn't understand why you eliminated towns with values 9998-9 - is that a data specific issue?